Bottom: edc91702d46d366c053da78ee019ea3cf43fd7eb
Top:    04a74d84954cc67eb428c6db07846c7da2416ab5
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 20:33:28 +0800

Refresh of vhost_net-split-out-datacopy

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 7788b27..5c3b9ef 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -700,13 +700,10 @@ static void handle_tx(struct vhost_net *net)
 	vhost_disable_notify(&net->dev, vq);
 	vhost_net_disable_vq(net, vq);
 
-	if (vhost_sock_zcopy(sock)) {
-		printk("zcopy!\n");
+	if (vhost_sock_zcopy(sock))
 		handle_tx_zerocopy(net, sock);
-	} else {
-		printk("data copy!\n");
+	else
 		handle_tx_copy(net, sock);
-	}
 
 out:
 	mutex_unlock(&vq->mutex);
